src/main/java/com/bugsnag/logback/DefaultLogEventAwareCallback.java

```diff
@@ -0,0 +1,72 @@
+package com.bugsnag.logback;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.bugsnag.Report;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+
+/** Default implementation that obtains data about the current user and additional tabs from MDC. */
+public class DefaultLogEventAwareCallback extends AbstractLogEventAwareCallback {
```

Comment:
Instead of "default", maybe adding "MDC" to the name is more accurate?

Reply:
Maybe we can call it just `MdcCallback`?

Reply:
Sure :)